### ISSUE TITLE
QA for Requests and Ideas, independent of QA of output

### ISSUE THREAD

**Issue by inverted-capital** [dreamcatcher-tech/dreamcatcher-tech.github.io#20]
We need to include a means to do QA on the quality of a Request or an Idea itself, in isolation.  This would allow it to enter the liquidity pools to able to be funded.  We must have this test to prevent garbage entering the pools, which will cause disputes when it comes to completion.

To QA the output of a funded Idea, we must first have had a good quality Request and Idea.  The Request must have been clear on what done means, and the Idea needs to be specific as to what it will deliver.  QA should also determine if a given Idea is a suitable candidate for the Requests that it proposes to solve.

Then QAs job upon delivery is only to determine if the presented output matches the Idea that was funded.

**Comment by TemperBead** [comment]
Agree on some points but not all.  Would like to relax the requirement for Requests to have "Done".  I think there's a spectrum of Requests from a dumb-ass idea just stuck out there to in effect a Request for Tender.

On the QA, yes.  And actually the important of this in all of the Requests we've raised from Yawp to Journo highlights the value of someone who checks, and who's reputation (and income stream) incentivises them to be disinterested, exact and honest.  

Have put some notes down locally here on this but have reached the end of my window for today, so will pick up tomorrow.

**Comment by inverted-capital** [comment]
Yes Requests should be able to exist without even asking for QA to pass them.  I think QA is related to a pool ?  All Requests that are included in a particular pool have to have passed a certain QAs standards ?
